dapps/onboarding.vue/src/components/mnemonic/mnemonic.ts

```diff
@@ -23,63 +23,63 @@ import { Prop } from 'vue-property-decorator';
 
 // evan.network imports
 import { EvanComponent } from '@evan.network/ui-vue-core';
-import * as dappBrowser from '@evan.network/ui-dapp-browser';
+import { Mnemonic } from '@evan.network/api-blockchain-core';
 
 @Component({ })
-export default class Mnemonic extends mixins(EvanComponent) {
+export default class MnemonicComponent extends mixins(EvanComponent) {
   /**
    * incoming mnemonic, if empty, input will automatically enabled
    */
   @Prop({
     type: String,
-    default: function() {
```

Comment:
how could the default be a function returning a string while the type should be a primitive string?
Same for disabled property.